pkg/policy/policy.go

```diff
@@ -123,6 +124,11 @@ func (p *Policies) Hash(kind string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("failed to load built-in / external policies: %s: %w", kind, err)
 	}
+	/*
+		if len(policies) == 0 {
+			return "", errors.New("no policies found")
+		}
```

Comment:
there is some problem with this check or integration test for it

Reply:
Hmm interesting - does it fail locally for you? In my local setup I've run it a few times without any failures.

Reply:
I removed it for now based on our discussion https://github.com/aquasecurity/trivy-operator/pull/2520/commits/4792da8af292525824162cb75d18b95ab79b5377